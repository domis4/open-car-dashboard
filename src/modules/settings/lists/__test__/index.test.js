import React from 'react'
import { shallow } from 'enzyme'
import List from '../index'

describe('settings list module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<List />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
