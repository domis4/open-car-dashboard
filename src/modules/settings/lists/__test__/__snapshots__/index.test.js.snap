// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`settings list module index.js expects component to match snapshot 1`] = `
<WithStyles(Paper)
  className="View-root-01"
>
  <WithStyles(Typography)
    component="h1"
  >
    Settings
  </WithStyles(Typography)>
  <WithStyles(Table)
    className="View-table-02"
  >
    <WithStyles(TableBody)>
      <WithStyles(TableRow)
        key="1"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          <span>
            <pure(AirplanemodeActiveIcon)
              className="View-icon-03"
            />
            Airplane Mode
          </span>
        </WithStyles(TableCell)>
        <WithStyles(TableCell)
          numeric={true}
        />
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="2"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          <span>
            <pure(NetworkWifiIcon)
              className="View-icon-03"
            />
            Wi-Fi
          </span>
        </WithStyles(TableCell)>
        <WithStyles(TableCell)
          numeric={true}
        />
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="3"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          <span>
            <pure(BluetoothIcon)
              className="View-icon-03"
            />
            Bluetooth
          </span>
        </WithStyles(TableCell)>
        <WithStyles(TableCell)
          numeric={true}
        />
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="4"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          <span>
            <pure(NetworkCellIcon)
              className="View-icon-03"
            />
            Cellular
          </span>
        </WithStyles(TableCell)>
        <WithStyles(TableCell)
          numeric={true}
        />
      </WithStyles(TableRow)>
      <WithStyles(TableRow)
        key="5"
      >
        <WithStyles(TableCell)
          component="th"
          scope="row"
        >
          <span>
            <pure(SettingsIcon)
              className="View-icon-03"
            />
            General
          </span>
        </WithStyles(TableCell)>
        <WithStyles(TableCell)
          numeric={true}
        />
      </WithStyles(TableRow)>
    </WithStyles(TableBody)>
  </WithStyles(Table)>
</WithStyles(Paper)>
`;
